parts/k8s/kubernetesconfigs.sh

```diff
@@ -190,6 +190,19 @@ configureCNI() {
     # needed for the iptables rules to work on bridges
     retrycmd_if_failure 120 5 25 modprobe br_netfilter || exit $ERR_MODPROBE_FAIL
     echo -n "br_netfilter" > /etc/modules-load.d/br_netfilter.conf
+    if [ "$IS_HOSTED_MASTER" = "true" ]; then
```

Comment:
Should this be squiggly-bracketed like the other env vars? `"${IS_HOSTED_MASTER}"`